examples/00_quick_start/sasrec-tf2.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>would it be possible to run the notebook and show the logs? in the rest of the notebooks we follow that pattern</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Added the logs. </p>